frontend/src/components/common/range-selector/range-selector.tsx

```diff
@@ -29,7 +29,12 @@ const RangeSelector: FC<Props> = (props) => {
   };
 
   useEffect(() => {
-    if (+maxValue && +minValue > +maxValue) {
+    setMinValue(props.selectedMin);
+    setMaxValue(props.selectedMax);
+  }, [props.selectedMin, props.selectedMax]);
+
+  useEffect(() => {
+    if (+minValue && +maxValue && +minValue > +maxValue) {
```

Comment:
Think how to simplify it

Reply:
I don't know how to simplify it. This check needs to swap values if min value higher tnan max, so we need to know if both values exists. For example minYear=2015, maxValue=2000 and in the next line they are swaped.

Reply:
@elador13 
I think that we can convert props to numbers when we assign 'em to the `useState` and in `useEffect`